package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "kara-proxy",
+  "version": "1.0.0",
+  "description": "charities / contributors",
+  "main": "src/index.jsx",
+  "scripts": {
+    "build": "webpack --config webpack.config.js --WATCH",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
With jest and some other testing dependencies installed, it probably makes sense to update this script to run jest.